config.properties

```diff
@@ -0,0 +1,3 @@
+#Sun Mar 11 00:19:47 EST 2018

```

Comment:
You added `config.properties` to `.gitignore` but then you commit it here. I believe that the either file should be removed or the `.gitignore` entry should be removed.

Reply:
The reason I did this is so the file exists when downloading the application. Values in the config.properties should not carry over to other installations. Therefore, I added it to the gitignore to prevent that. Will this be an issue?

Reply:
I went ahead and removed the config.properties file because it will be automatically created on the first program execution.